src/main/java/seedu/address/model/Model.java

```diff
@@ -82,12 +82,25 @@ public interface Model {
      */
     boolean hasMeeting(Meeting meeting);
 
+    /**
+     * Returns true if a meeting with the same date, time, and description as {@code meeting} exists in the MyInsuRec.
+     * This is a stronger version of hasMeeting.
+     */
+    boolean hasSpecificMeeting(Meeting meeting);
```

Comment:
im not sure/ dun think this is necessary a stronger version? in that now that is only rejecting the meeting if meeting same time AND client same name AND same description, this means the checks got more lenient and if same time but diff description, they can actl get in